playground/PostgresEndToEnd/PostgresEndToEnd.ApiService/PostgresEndToEnd.ApiService.csproj

```diff
@@ -8,6 +8,9 @@
 
   <ItemGroup>
     <AspireProjectOrPackageReference Include="Aspire.Npgsql.EntityFrameworkCore.PostgreSQL" />
+    <!-- Npgsql EF needs to match the same major version as the underlying Npgsql assemblies. -->
+    <PackageReference Include="Npgsql.DependencyInjection" VersionOverride="$(Npgsql8Version)" />
```

Comment:
NIT: Should this be conditioned? Mainly thinking that at some point DefaultTargetFramework will change so this would likely break.

Reply:
Adding a TFM condition on a project that only targets 1 TFM isn't useful IMO. When we change DefaultTargetFramework (which probably won't be until net10 or beyond), we can adjust the references.